paasta_tools/cli/cmds/status.py

```diff
@@ -785,9 +797,42 @@ def _print_flink_status_from_job_manager(
         raise ValueError(f"expected config sha on Flink, but received {metadata}")
     if config_sha.startswith("config"):
         config_sha = config_sha[6:]
-
     output.append(f"    Config SHA: {config_sha}")
 
+    if verbose:
+        # Print Flink Pool information
+        flink_pool = None
+        if flink_instance_config is not None:
+            flink_pool = flink_instance_config.get_pool()
+        output.append(f"    Flink Pool: {flink_pool}")
+
+        # Print ownership information
+        flink_monitoring_team = None
+        if flink_instance_config is not None:
+            flink_monitoring_team = flink_instance_config.get_team()
+        if flink_monitoring_team is None:
+            flink_monitoring_team = get_team(
+                overrides={}, service=service, soa_dir=DEFAULT_SOA_DIR
+            )
+        output.append(f"    Owner: {flink_monitoring_team}")
+
+        # Print rb information
+        flink_rb_for_instance = None
+        if flink_instance_config is not None:
+            flink_rb_for_instance = flink_instance_config.get_runbook()
+            if flink_rb_for_instance is None:
+                flink_rb_for_instance = get_runbook(
+                    overrides={}, service=service, soa_dir=DEFAULT_SOA_DIR
+                )
+        output.append(f"    Flink Runbook: {flink_rb_for_instance}")
+
+        # Print Flink repo links
+        output.append(f"    Repo(git): https://github.yelpcorp.com/services/{service}")
+        output.append(
+            f"    Repo(sourcegraph): https://sourcegraph.yelpcorp.com/services/{service}"
+        )
```

Comment:
according to the type hints, `flink_instance_config` will never be None - so we can probably unindent things a bit here :)
```suggestion
    if verbose:
        # Print Flink Pool information
        flink_pool = flink_instance_config.get_pool()
        output.append(f"    Flink Pool: {flink_pool}")

        # Print ownership information
        flink_monitoring_team = flink_instance_config.get_team() or get_team(
            overrides={}, service=service, soa_dir=DEFAULT_SOA_DIR
        )
        output.append(f"    Owner: {flink_monitoring_team}")

        # Print rb information
        flink_rb_for_instance = flink_instance_config.get_runbook() or get_runbook(
            overrides={}, service=service, soa_dir=DEFAULT_SOA_DIR
        )
        output.append(f"    Flink Runbook: {flink_rb_for_instance}")

        # Print Flink repo links
        output.append(f"    Repo(git): https://github.yelpcorp.com/services/{service}")
        output.append(
            f"    Repo(sourcegraph): https://sourcegraph.yelpcorp.com/services/{service}"
        )
```

Reply:
(probably needs some slight reformatting after that refactor since i did that in the comment editor :p)

Reply:
i think we can also clean up the monitoring_tools parts a bit more later so that we can have a single place to load from monitoring.yaml or the instance overrides - but this is probably fine for now

Reply:
Sure, will pick up in the next refactor PR